packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -51,6 +55,35 @@ class CpuProfilerController with SearchControllerMixin<CpuStackFrame> {
 
   Iterable<String> get userTags => _dataByTag[userTagNone]?.userTags ?? [];
 
+  /// The toggle filters available for the CPU profiler.
+  ///
+  /// This is a getter so that `serviceManager` is initialized by the time we
+  /// access this list.
+  // TODO(kenz): make this lazy once we migrate to null safety.
+  List<ToggleFilter> get toggleFilters => _toggleFilters ??= [
+        ToggleFilter<CpuStackFrame>(
+          name: 'Hide Native code',
+          includeCallback: (stackFrame) => stackFrame.url.isNotEmpty,
+        ),
+        ToggleFilter<CpuStackFrame>(
+          name: 'Hide core Dart libraries',
+          includeCallback: (stackFrame) =>
+              !stackFrame.url.contains('org-dartlang-sdk'),
```

Comment:
can this be more specific? Would be nice to make this 
url.startsWith instead of contains and would be good to have a link to a bug tracking why these urls show as org-dartlang-sdk instead of `dart:`?

Reply:
Is this something we should be simplifying on our end (like we do currently for package/flutter urls). For example, 
`org-dartlang-sdk:///third_party/dart/sdk/lib/_internal/vm/lib/compact_hash.dart` -> `dart:sdk/lib/_internal/vm/lib/compact_hash.dart`

WDYT

Reply:
what do we show if you open compact_hash.dart in the debugger page? It would be nice to keep it consistent with that.
I imagine it should ideally be
`dart:core/compact_hash.dart`
although I doubt we surface it that way.

Reply:
you are right, in the debugger we show `dart:collection-patch/compact_hash.dart`. Switched to doing this and pre-processing urls for stack frames this way.

Reply:
we are doing this in DevTools for now and I filed an issue for the VM to send package uris instead of file path uris https://github.com/dart-lang/sdk/issues/46872